gatsby/content/projects/bots/matrix-eno-bot.mdx

```diff
@@ -27,9 +27,10 @@ the template and the SDK are all written in Python 3. The commands that the
 matrix-eno-bot comes with are written in bash and in python3.
 
 Intended audience/users:
-  small homeserver set up for friends and family or small community
-  tinkerers who want to learn how a bot works
-  people who want to play around with Python code and Matrix
+
+    * small homeserver: set up for friends and family or small community
```

Comment:
this will render like

```
     * small homeserver: set up for friends and family or small community
```

if you remove the leading spaces it will render like

* small homeserver: set up for friends and family or small community

which I guess is preferable

Reply:
Well, you are absolutely correct. Looks better. Thanks for the help.
Kind regards! 

Changes committed in https://github.com/matrix-org/matrix.org/pull/867/commits/a48e6fc757fc7aecd0c575986c73425f0e57745f